src/interp/interp.cc

```diff
@@ -1056,25 +1097,30 @@ Result Thread::ElemDrop(const uint8_t** pc) {
 
 Result Thread::TableCopy(const uint8_t** pc) {
   Table* table = ReadTable(pc);
-  uint32_t table_size = table->func_indexes.size();
   uint32_t size = Pop<uint32_t>();
   uint32_t src = Pop<uint32_t>();
   uint32_t dst = Pop<uint32_t>();
   bool copy_backward = src < dst && dst - src < size;
-  bool ok = ClampToBounds(dst, &size, table_size);
+  bool ok = ClampToBounds(dst, &size, table->size());
   // When copying backward, if the range is out-of-bounds, then no data will be
   // written.
   if (ok || !copy_backward) {
-    ok &= ClampToBounds(src, &size, table_size);
+    ok &= ClampToBounds(src, &size, table->size());
     if (size > 0) {
-      Index* data = table->func_indexes.data();
-      memmove(data + dst, data + src, size * sizeof(Index));
+      Ref* data = table->entries.data();
+      memmove(data + dst, data + src, size * sizeof(table->entries[0]));
```

Comment:
Why not `sizeof(Ref)`?

Reply:
Done.  I was trying to make the code robust against the type of the table entries changing .. perhaps `sizeof(data[0])` would be more clear?